projects/notebooks/roles/notebooks_dashboard_scale_test/templates/ods-ci_job.yaml

```diff
@@ -51,6 +51,26 @@ spec:
           value: "{{ notebooks_dashboard_scale_test_user_index_offset }}"
         - name: REDIS_SERVER
           value: "{{ notebooks_dashboard_scale_test_state_signal_redis_server }}"
+        - name: DASHBOARD_PRODUCT_NAME
+          value: "{{ rhods_dashboard_product_name }}"
+        - name: S3_BUCKET_NAME
+          value: "{{ notebooks_dashboard_scale_test_minio_bucket_name }}"
+        - name: MINIO_NAMESPACE
+          value: "{{ notebooks_dashboard_scale_test_minio_namespace }}"
+        - name: S3_SECRET_KEY
+          valueFrom:
+            secretKeyRef:
+              name: minio
+              key: MINIO_S3_SECRET_KEY
```

Comment:
I don't really like this, if in the script we put the `env` command, the secret key is leaked :/ 
We'll need to rediscuss this

Reply:
should be passed within the variables.yaml something file, which already handles secret values

Reply:
understood,  removed secret from the env and updated the secret with TESTUSER password.